docs/source/trivy.rst

```diff
@@ -0,0 +1,89 @@
+#####
+trivy
+#####
+
```

Comment:
Can you give a few words of what trivy is, and how does it fit the ecosystem with kube-hunter, etc?

Reply:
I would also mention two other things:
- it's in go
- how to get started (links to onboarding).

Reply:
I would also mention the fact there is a github action for it :)

Reply:
> It's in go

I don't think it's really relevant . Plus, you can take a look at this fact by checking the provided links or the OBS package.

> how to get started (links to onboarding).

Will do

> I would also mention the fact there is a github action for it :)

Already documented

Reply:
> I don't think it's really relevant . Plus, you can take a look at this fact by checking the provided links or the OBS package.

Added it anyways :stuck_out_tongue_winking_eye: